ci/do_ci.ps1

```diff
@@ -11,8 +11,8 @@ $nproc = (Get-ComputerInfo).CsNumberOfLogicalProcessors
 $SRC_DIR = (Get-Item -Path ".\").FullName
 
 # Workaround https://github.com/bazelbuild/bazel/issues/18683
-$BAZEL_STARTUP_OPTIONS = "--output_base=C:\Out"
-$BAZEL_OPTIONS = "--copt=-DENABLE_ASYNC_EXPORT"
```

Comment:
this continues to be a wild hack. I linked the new path issue in the upstream issue, it should be improved once grpc moves its deps into bzlmod, but right now the paths are artificially longer. absolutely wild stuff.